lib/main.dart

```diff
@@ -128,4 +130,120 @@ class _HomePageState extends State<HomePage> {
       },
     );
   }
+
+  void _routeToInfo(ObjectInfo picData) {
+    setState(() {
+      Navigator.push(
+        context,
+        MaterialPageRoute(builder: (context) => ArtworkInfoScreen(picData)),
+      );
+    });
+  }
+}
+
+class ArtworkInfoScreen extends StatelessWidget {
+  final ObjectInfo picData;
+
+  ArtworkInfoScreen(this.picData);
+
+  @override
+  Widget build(BuildContext context) {
+    final imageData = (picData.images == null) ? null : picData.images[0];
+    final artistData = (picData.people == null) ? null : picData.people[0];
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Artwork Information'),
+      ),
+      body: SafeArea(
+        child: Column(
+          children: <Widget>[
+            Expanded(
+              child: ListView(
+                padding: const EdgeInsets.all(8.0),
+                children: <Widget>[
+                  Hero(
+                    tag: picData.primaryImageUrl,
+                    child: Image.network(picData.primaryImageUrl),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(bottom: 15.0, top: 15.0),
+                    child: Wrap(
+                      alignment: WrapAlignment.center,
+                      children: [
+                        Text(
+                          '${picData.title} (${picData.date})',
+                          style: TextStyle(
+                              fontSize: 24.0,
```

Comment:
What do you think about using a TextTheme? It could create a simplified approach to setting sizes and style (including in your other classes below so you don't have to provide text sizes) and also make it consistent across the whole app like the color choices in ThemeData. Up to you!

https://flutter.dev/docs/cookbook/design/themes

Reply:
I originally did this because I thought it would make my code simpler but it turns out since there are many small tweaks to each text style it end up getting in the way and for what I am doing for my presentation it actually saves me time to write it manually. Do you know of a way around this at all?